attributes/response/cddl/3_proof.cddl

```diff
@@ -0,0 +1,17 @@
+; Proof.  If requested, this is the set of steps constituting a proof of existence of what was requested in the state of the Merkle tree.
+proof@response-attribute-arg = [ + ( node-hash / key-value-hash / key-value-pair / parent / child ) ]
```

Comment:
The response attribute should be a map, it should contain at least 2 values:

1. the root hash, and
2. the proof itself.

Add an optional block height for now, but make it a attribute-related-index.

```suggestion
proof@response-attribute-arg = {
    ; Root application hash.
    0 => bstr,
    
    ; The proof operations.
    1 => proof,
    
    ; Extensible attribute related indices for extra information (implementation specific)
    * attribute-related-index => (),
}

proof = [ + ( node-hash / key-value-hash / key-value-pair / parent / child ) ]
```

And then in a separate CDDL `3_proof/0_blockchain.cddl` file, get the following in:

```cddl
proof@response-attribute-arg //= {
    ; Block height.
    [3, [0, 0]] => uint
}
```